skimage/segmentation/manual_segmentation.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib.patches import Polygon
+from matplotlib.collections import PatchCollection
+
+
+def manual(image, speed_up=1):
+    """Return a binary image based on the selections made with mouse clicks.
+
+    Parameters
+    ----------
+    image : (M, 2) array
```

Comment:
`(M, N[, 3])`: it is a 2-dimensional object, with dimensions of size M and N, and optionally a third (channels) dimension.